x/ibc/20-transfer/types/packet.go

```diff
@@ -0,0 +1,68 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// TransferPacketData defines a struct for the packet payload
+type TransferPacketData struct {
+	Denomination string  `json:"denomination" yaml:"denomination"`
+	Amount       sdk.Int `json:"amount" yaml:"amount"`
```

Comment:
ditto, use `sdk.Coins`

Reply:
About the `TransferPacketData`, it's defined in https://github.com/cosmos/ics/blob/master/spec/ics-020-fungible-token-transfer/README.md#data-structures.

The ICS is a universal standard, so we should avoid using SDK methods? @cwgoes